package.json

```diff
@@ -2,28 +2,21 @@
   "name": "third-party-capital",
   "version": "1.0.26",
   "description": "",
-  "main": "./lib/cjs/index.js",
-  "types": "./lib/cjs/types/index.d.ts",
-  "files": [
-    "lib/**/*"
-  ],
+  "type": "commonjs",
```

Comment:
I'd recommend exporting as a module by default

Reply:
worked after switching jest.config.js to jest.config.ts